packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -166,12 +166,11 @@ class Attributes {
         );
     }
 
-    public detectedErroneousAttribute(attributeId: CoreId | string, message?: string): ApplicationError {
-        let msg = `Detected an error regarding attribute '${attributeId.toString()}'.`;
-        if (typeof message !== "undefined") {
-            msg += ` ${message}`;
-        }
-        return new ApplicationError("error.runtime.attributes.detectedErroneousAttribute", msg);
+    public noOtherVersionOfIdentityAttributeHasBeenSharedWithPeerBefore(repositoryAttributeId: CoreId | string, peer: CoreAddress | string): ApplicationError {
+        return new ApplicationError(
+            "error.runtime.attributes.noOtherVersionOfIdentityAttributeHasBeenSharedWithPeerBefore",
+            `No other version of identity attribute '${repositoryAttributeId.toString()}' has been shared with peer '${peer.toString()}' before. If you wish to execute an initial sharing of this attribute, use 'ShareIdentityAttribute'.`
```

Comment:
IdentityAttribute ?

Reply:
Is your comment referring to the fact, that we use the term IdentityAttribute instead of RepositoryAttribute here (which we want to change in #9)? Or is it a suggestion to use the the writing IdentityAttribute in the error message instead of identity attribute? Currently, most error message use this writing for Attributes, but @britsta is working on a ticket, which among other things takes care of a consistence writing.

Reply:
The latter, but you're right - we should be consistent throughout the error messages and docs in terms of the cases of attributes (or Attributes? ;))